src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -309,7 +311,7 @@ node_future<> InternalNode::apply_child_split(
     return InternalNode::allocate(
         c, impl->field_type(), impl->is_level_tail(), impl->level());
   }).safe_then([this_ref, this, c, left_key, left_child, right_child,
-                insert_pos, insert_stage, insert_size](auto fresh_right) mutable {
+                insert_pos, insert_stage=insert_stage, insert_size=insert_size](auto fresh_right) mutable {
```

Comment:
Could you elaborate what's the warning here?

Reply:
it was an error

```
Errors: capturing structured-binding "special ref"
```

Reply:
What is a "special ref"? It looks to me `insert_stage` and `insert_size` are plain integers.

Reply:
no, there are not. see the discussion at https://stackoverflow.com/questions/46114214/lambda-implicit-capture-fails-with-variable-declared-from-structured-binding .